src/main/java/org/jabref/gui/actions/OldCommandWrapper.java

```diff
@@ -0,0 +1,50 @@
+package org.jabref.gui.actions;
+
+import javafx.beans.property.ReadOnlyDoubleProperty;
+
+import org.jabref.gui.BasePanel;
+import org.jabref.gui.util.BindingsHelper;
+
+import de.saxsys.mvvmfx.utils.commands.CommandBase;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+/**
+ * This wraps the old Swing commands so that they fit into the new infrastructure.
+ * In the long term, this class should be removed.
+ */
+public class OldCommandWrapper extends CommandBase {
+
+    private static final Log LOGGER = LogFactory.getLog(OldCommandWrapper.class);
```

Comment:
Please use the new logging     private static final Logger LOGGER = LoggerFactory.getLogger(OldCommandWrapper.class);

Reply:
Your PR is not yet merged into `javafxTable` and I prefer to do this as soon as all the other maintable PRs are merged

Reply:
Okay, just as a hint to make work then easier. Slf4j has a nice migrator tool (inlcuded in the zip download) https://www.slf4j.org/migrator.html to replace all occurences. Used that for the migration, too.